src/plugins/telemetryTable/components/table.vue

```diff
@@ -138,6 +138,13 @@
             class="c-telemetry-table__drop-target"
             :style="dropTargetStyle"
         ></div>
+
+        <progress-bar
+            v-if="progressLoad.progressPerc !== 100"
```

Comment:
I think if we change this to `v-if="loading"` then we should be good.

Reply:
Made change per suggestion and pushed.